docs/DebugView.md

```diff
@@ -25,3 +25,55 @@ await debugConfiguration.selectLaunchConfiguration("Test Launch");
 // start selected launch configuration
 await debugView.start();
 ```
+
+### Sections
```

Comment:
I like that you have added all supported sections . 🚀 

Could we please also add detailed screenshots for each section with some items there? to better showcase the usecase? 🙂

Reply:
added